package/CHANGELOG

```diff
@@ -13,18 +13,20 @@ The rules for this file:
   * release numbers follow "Semantic Versioning" http://semver.org
 
 ------------------------------------------------------------------------------
-mm/dd/17 richardjgowers, rathann, orbeckst
+mm/dd/17 richardjgowers, rathann, orbeckst, kain88-de, tyler.je.reddy
```

Comment:
Oh well, just merged #503 with `tylerjereddy` – @tylerjereddy you need to say what you want.